dev/com.ibm.ws.microprofile.reactive.messaging.kafka.adapter/src/com/ibm/ws/microprofile/reactive/messaging/kafka/adapter/KafkaProducer.java

```diff
@@ -18,10 +18,12 @@
 public interface KafkaProducer<K, V> extends KafkaAdapter {
 
     /**
-     * @param record
+     * @param topic
+     * @param channelName
+     * @param value
      * @param callback
      */
-    void send(String topic, V value, Callback callback);
+    void send(String topic, String channelName, V value, Callback callback);
```

Comment:
This method in particular needs doc because it's not a simple passthrough.

Reply:
it is back to being passthrough